src/persist-client/src/internal/state.rs

```diff
@@ -1154,31 +1177,6 @@ where
         ret
     }
 
-    pub fn handles_needing_expiration(
-        &self,
-        now_ms: EpochMillis,
-    ) -> (Vec<LeasedReaderId>, Vec<WriterId>) {
-        let mut readers = Vec::new();
-        for (reader, state) in self.collections.leased_readers.iter() {
-            let time_since_last_heartbeat_ms =
-                now_ms.saturating_sub(state.last_heartbeat_timestamp_ms);
-            if time_since_last_heartbeat_ms > state.lease_duration_ms {
-                readers.push(reader.clone());
-            }
-        }
-        // critical_readers don't need forced expiration (in fact, that's the
```

Comment:
just making sure we dropped this comment about critical readers on purpose (fine, if so)

Reply:
I didn't have any particular intent one way or the other, so added it back for now; if we decide to clear out comments that can be a separate change.